examples/ratelimit/04-contour-config.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
Not sure if we should include this or just reference what to setup. I guess it's a nice copy/paste way to follow a guide so probably good.